.circleci/scripts/build-package

```diff
@@ -3,6 +3,20 @@ set -o errexit  \
     -o nounset  \
     -o pipefail
 
+REGEX_RELEASE_VERSION='[[:digit:]]+\.[[:digit:]]\.[[:digit:]]'
```

Comment:
Are we okay that this won't allow double digit minor/patch versions?

Should it be this instead:
`[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+`